api/v1/installation_types.go

```diff
@@ -474,6 +474,17 @@ type CalicoNetworkSpec struct {
 	// Sysctl configures sysctl parameters for tuning plugin
 	// +optional
 	Sysctl []Sysctl `json:"sysctl,omitempty"`
+
+	// PolicySetupTimeoutSeconds delays new pods from going ready
```

Comment:
"going ready" should probably be "running containers" - we actually prevent the code from running altogether until the policy is programmed.

Reply:
How about "[...] from entering the Running phase" ?

Reply:
I think "entering the running phase" is too vague - it doesn't drive the point home that the users application code won't even attempt to be launched. 

"from running" is OK, but IMO still not precise - the pod itself will be running (as in, the pod will have established a network namespace with an  ID and an IP address) - what won't be running are the containers within the pod.

Reply:
Updated to "running containers" now